source/online_usecases/validation_framework/validation_framework.rst

```diff
@@ -184,7 +184,26 @@ We currently have the following Use Cases on the Collaboratory:
 
     Some of the features currently included are soma's diameter and the maximal branch order in the dendrites, besides the number of trunk sections, -X,Y,Z- extents, field's diameter and total path-length of both the axon and the dendrites.
 
-    **Note:** Currently only FS interneurons can be considered, since observation data is missing for other neuron types.
+    **Note:** Currently only Striatum Fast-Spiking Interneurons (FSI) can be considered, since observation data is missing for other neuron types.
+
+**6) Basal Ganglia Population Morphology Validation**
+  The average morphometrics of a population of (digitally reconstructed) Fast-Spiking Interneurons (FSI) in Striatum, is validated against experimental data. Additional plots are provided to visualize some statistics derived from the morphometrics of the indiviual cells, e.g. linear regression analysis, histograms and Kernel-Distribution-Estimates (KDE) for single features, and bi-dimensionl joint KDEs for pairs of uncorrelated features.
+
+  This test shall take as input a directory containing neuronal morphologies. The features extraction is carried out using 'NeuroM' (https://github.com/BlueBrain/NeuroM), so the test currently supports only 'NeuroM' compatible formats.
+
+  The user decides whether to run the validations for all available morphologies, or a subset of these. The validation test evaluates the morphology in two stages:
+
+  - **Hard Constraints** 
+    Here we evaluate the integrity of the neuronal reconstruction in order to determine if it is appropriate for further evaluations. The evaluations here can be sub-divided into the following NeuroM features (apps): - morph_check - cut_plane_detection
+
+  - **Soft Constraints** *[Currently only available for Fast Spiking Interneurons]*
+    Neuronal reconstructions that pass the 'Hard Constraints' are evaluated here for their morphometric features. The features are extracted using NeuroM's morph_stats app, wherever possible, either directly or as a combination of multiple features. The average morphometrics of the population of neurons is then computed. Those mean values are then compared against experimentally obtained data, as determined by the particular validation test being executed.
+
+    Some of the features currently included are soma's diameter and the maximal branch order in the dendrites, besides the number of trunk sections, -X,Y,Z- extents, field's diameter and total path-length of both the axon and the dendrites.
+
+    Additional plots are provided to visualize some statistics derived from the morphometrics of the indiviual cells, e.g. linear regression analysis, histograms and Kernel-Distribution-Estimates (KDE) for single features, and bi-dimensionl joint KDEs for pairs of uncorrelated features.
```

Comment:
indiviual cells -> individual cells
dimensionl -> dimensional ?

Reply:
It is corrected, thanks